releases.yml

```diff
@@ -1,4 +1,24 @@
 releases:
+  art3657:
+    assembly:
+      type: custom
+      basis:
+        assembly: "4.9.13"
+        "reference_releases!": {}
+    group:
+      advisories:
+        extras: -1
+        image: -1
+        metadata: -1
+        rpm: -1
```

Comment:
this is optional, can we just leave it out?
```suggestion
```

Reply:
I followed it from [here](https://github.com/openshift-eng/art-docs/blob/master/assemblies.md#building-rhcos-with-new-dependencies), as I understand the intent is to unlink the inherited advisories

Reply:
oh; right.